src/sourmash/cli/tax/classify.py

```diff
@@ -0,0 +1,68 @@
+"""classify genomes from gather results"""
+
+import argparse
+import sourmash
+from sourmash.logging import notify, print_results, error
+from sourmash.cli.utils import add_threshold_arg
+
+def subparser(subparsers):
+    subparser = subparsers.add_parser('classify')
```

Comment:
please add `help=...` information here, so that `sourmash tax classify` output is more informative -

currently, it's
```
% sourmash tax classify
usage:  classify [-h] [-g [GATHER_CSV [GATHER_CSV ...]]] [--from-file FILE]
                 [-q] -t FILE [FILE ...] [-o OUTPUT_BASE]
                 [-r {strain,species,genus,family,order,class,phylum,superkingdom}]
                 [--keep-full-identifiers] [--keep-identifier-versions]
                 [--fail-on-missing-taxonomy]
                 [--output-format {summary,krona} [{summary,krona} ...]] [-f]
                 [--containment-threshold CONTAINMENT_THRESHOLD]
 classify: error: the following arguments are required: -t/--taxonomy-csv